care.js

```diff
@@ -243,6 +244,7 @@ function makeGraphBox(label) {
   options.barWidth= 5;
   options.xOffset= 4;
   options.maxHeight= 10;
+  options.barFgColor= 'black';
```

Comment:
What is this `barFgColor`option used for? If it is not used could you please remove it, Thx!